### ISSUE TITLE
Cannot download the raw zip model files.

### ISSUE THREAD

**Issue by riple** [Microsoft/ELL-models#57]
All the model files in .zip format cannot be downloaded. All I got from the RAW link on the .zip file page is a 133Byte description of the file property.

I even tried to download the whole repository in a zip package. The content of the particular *.zip file is the same 133Byte description.

**Comment by riple** [comment]
I even tried to git clone the whole repository to my computer, and the *.zip files are all broken. 
All the other file formats are usable, such as .svg and .txt.

**Comment by riple** [comment]
I found out that the .zip files are stored as Git LFS files. So I tried the "git lfs install".
And this is what it said:

> desktop:~/work/ell/ELL-models/models/ILSVRC2012/d_I224x224x3CMCMCMCMCMCMC1AS$ git lfs fetch
fetch: Fetching reference refs/heads/master
batch response: This repository is over its data quota. Purchase more data packs to restore access.                                           
batch response: This repository is over its data quota. Purchase more data packs to restore access.
error: failed to fetch some objects from 'https://github.com/Microsoft/ELL-models.git/info/lfs'
